plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,2305 @@
+        description: |
+          Brightness level of the AP's LED. Accepts values from 1 to 8.
+          Example: led_brightness_level: 3
+        type: int
+        required: False
+      ap_mode:
+        description: |
+          Mode of operation for the Access Point (AP). Possible values include "local/flexconnect" or "monitor" or "sniffer" or Bridge/Flex+Bridge.
+          - If Ap mode is local/flex, then only radio role assignment can be changed.
+          Example: ap_mode: "local"
+        type: str
+        required: False
+      location:
+        description: |
+          Location name of the AP. Provide this data if a change is required.
+          Example: location: "Bangalore"
+        type: str
+        required: False
+      failover_priority:
+        description: |
+          Priority order for failover in AP configuration. Accepts "Low" or "Medium" or "High" or "Critical".
+        type: str
+        required: False
+      clean_air_si_2.4ghz/5ghz/6ghz:
+        description: |
+          Clean Air Spectrum Intelligence (SI) feature status for 2.4/5/6GHz band. Indicates whether Clean Air Spectrum Intelligence is enabled or disabled.
+          Example: clean_air_si_2.4ghz: "Enabled"
+        type: str
+        required: False
+      primary_controller_name:
+        description: |
+          Name or identifier of the primary wireless LAN controller (WLC) managing the Access Point (AP).
+          Example: primary_controller_name: "SJ-EWLC-1"
+        type: str
+        required: False
+      primary_ip_address:
+        description: |
+          IP address of the primary wireless LAN controller (WLC) managing the Access Point (AP).
+          Example: primary_ip_address:
+                   address: "204.192.4.200"
+        type: str
+        required: False
+      secondary_controller_name:
+        description: |
+          Name or identifier of the secondary wireless LAN controller (WLC) managing the Access Point (AP).
+          Example: secondary_controller_name: "Inherit from site/Clear"
+        type: str
+        required: False
+      secondary_ip_address:
+        description: |
+          IP address of the secondary wireless LAN controller (WLC) managing the Access Point (AP).
+          Example: secondary_ip_address:
+                   address: "10.0.0.2"
+        type: str
+        required: False
+      tertiary_controller_name:
+        description: |
+          Name or identifier of the tertiary wireless LAN controller (WLC) managing the Access Point (AP).
+          Example: tertiary_controller_name: "Clear"
+        type: str
+        required: False
+      tertiary_ip_address:
+        description: |
+          IP address of the tertiary wireless LAN controller (WLC) managing the Access Point (AP).
+          Example: tertiary_ip_address:
+                   address: "10.0.0.3"
+        type: str
+        required: False
+      2.4ghz_radio/5ghz_radio/6ghz_radio/xor_radio/tri_radio:
```

Comment:
Same here as well.. Either we can make it sub-options or separate them out. Can we change powerlevel into power_level?

```
radio_settings:
  description: Configuration options for radio interfaces.
  type: dict
  required: False
  suboptions:
    2_4ghz:
      description: Configuration options for the 2.4GHz radio interface.
      type: dict
      required: False
      suboptions:
        admin_status:
          description: |
            Administrative status for the 2.4GHz radio interface.
            Example: admin_status: "Enabled"
          type: str
          required: False
        antenna_name:
          description: |
            Name or type of antenna used for the 2.4GHz radio interface.
            Example: antenna_name: "other"
          type: str
          required: False
        antenna_gain:
          description: |
            Antenna gain value in decibels (dB) for the 2.4GHz radio interface.
            Example: antenna_gain: 4
          type: int
          required: False
        radio_role_assignment:
          description: |
            Role assignment mode for the 2.4GHz radio interface. Accepts "Auto" or "Client-serving" or "Monitor".
            - If radio_role_assignment is "client-serving", then only power-level and channel-level can be changed.
            Example: radio_role_assignment: "Auto"
          type: str
          required: False
        cable_loss:
          description: |
            Cable loss in dB for the 2.4GHz radio interface.
            Example: cable_loss: 75
          type: int
          required: False
        antenna_cable_name:
          description: |
            Name or type of antenna cable used for the 2.4GHz radio interface.
            Example: antenna_cable_name: "other"
          type: str
          required: False
        channel_assignment_mode:
          description: |
            Mode of channel assignment for the 2.4GHz radio interface. Accepts "Global" or "Custom".
            - For 2.4GHz Custom, it accepts values like 1 to 12.
            Example: channel_assignment_mode: "Custom"
          type: str
          required: False
        channel_number:
          description: |
            Custom channel number configured for the 2.4GHz radio interface.
            Example: channel_number: 6
          type: int
          required: False
        power_assignment_mode:
          description: |
            Mode of power assignment for the 2.4GHz radio interface. Accepts "Global" or "Custom".
            - In Custom, it accepts values 1 to 8.
            Example: power_assignment_mode: "Custom"
          type: str
          required: False
        power_level:
          description: |
            Custom power level configured for the 2.4GHz radio interface.
            Example: powerlevel: 3
          type: int
          required: False
    5ghz:
      description: Configuration options for the 5GHz radio interface.
      type: dict
      required: False
      suboptions:
        admin_status:
          description: |
            Administrative status for the 5GHz radio interface.
            Example: admin_status: "Enabled"
          type: str
          required: False
....
   
```

Reply:
Updated for all radio interface